front/src/modules/pathfinding/components/Itinerary/ModalSuggestedVias.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useCallback } from 'react';
+
+import { Dash, Plus, Trash } from '@osrd-project/ui-icons';
+import cx from 'classnames';
+import { compact } from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+
+import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
+import ModalBodySNCF from 'common/BootstrapSNCF/ModalSNCF/ModalBodySNCF';
+import ModalFooterSNCF from 'common/BootstrapSNCF/ModalSNCF/ModalFooterSNCF';
+import ModalHeaderSNCF from 'common/BootstrapSNCF/ModalSNCF/ModalHeaderSNCF';
+import { useOsrdConfActions, useOsrdConfSelectors } from 'common/osrdContext';
+import { isVia } from 'modules/pathfinding/utils';
+import type { SuggestedOP } from 'modules/trainschedule/components/ManageTrainSchedule/types';
+import { useAppDispatch } from 'store';
+import { formatUicToCi } from 'utils/strings';
+
+type ModalSuggestedViasProps = {
+  removeAllVias: () => void;
+  pathProperties: ManageTrainSchedulePathProperties;
```

Comment:
Directly give the suggestedOps to the modal
```suggestion
  suggestedOps: ManageTrainSchedulePathProperties['suggestedOperationalPoints'];
```

Reply:
Done